tests/integration-tests/tests/ad_integration/test_ad_integration/test_ad_integration/ad_stack.yaml

```diff
@@ -190,28 +194,39 @@ Resources:
                 # Create users in directory
                 # TODO: add ability to trigger this via lambda
                 for NEW_USER_ALIAS in \${NEW_USER_ALIASES}; do
-                    echo "Creating \${NEW_USER_ALIAS} in directory \${DIRECTORY_ID}"
-                    /usr/local/bin/add_user.sh \${NEW_USER_ALIAS}
+                  MAX_ATTEMPTS=5
+                  for ATTEMPT in \$(seq 1 \${MAX_ATTEMPTS}); do
+                    echo "Creating user \${NEW_USER_ALIAS} (attempt \${ATTEMPT}/\${MAX_ATTEMPTS})"
```

Comment:
how can we identify variables requiring the escape characters from the others?
We should find an internal convention (e.g. lowercase vs uppercase, or underscore prefix)...

Reply:
The ones requiring the escape are those within EOF sections whose value is determined at runtime.

Reply:
We agreed on having a convention for such variables at the next iteration on this part